package.json

```diff
@@ -22,13 +22,14 @@
     "react-helmet": "^6.1.0"
   },
   "devDependencies": {
+    "@types/react-helmet": "^6.1.0",
     "contentful-import": "^7.9.38",
     "netlify-cli": "^3.10.1"
   },
   "engines": {
     "node": ">=14.16.0",
     "npm": "please-use-yarn",
-    "yarn": ">= 1.22.5"
+    "yarn": ">= 1.22"
```

Comment:
I noticed that the Netlify builds were failing because of a minor version mismatch. This should fix it.